.github/stale.yml

```
@@ -0,0 +1,15 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: "0 0 * * *" # Will be triggered every day at midnight UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
```

hash instead of version here maybe?
```suggestion
      - uses: actions/stale@asdfasdfasdf # v6
```